requirements.txt

```diff
@@ -1,4 +1,12 @@
-flask
```

Comment:
I shall fix it. I am not that familiar with these packages so I just used `pip freeze` to do it.